_documentation/05-User-Manual.md

```diff
@@ -14,19 +14,126 @@ The user is relieved from most of the complexities as compared to the manufactur
 Further, many manufacturers have options that cannot be translated into IPP attributes. So the users have the option to change their printer/scanner properties through the Web GUI provided with Printer/Scanner Application.
 
 
-## Installing Applications
+## Installation
 
 For installing Printer/Scanner Applications, you should have `snap` installed on your system.
 
+### Snap
+
 To install `snap` from the Software Manager application, search for `snapd` and click Install.
 
 Alternatively, `snapd` can be installed from the command line:
 
     sudo apt install snapd
 
-Once `snap` is installed, you can easily install any driver with ease. Find the name of the application compatible with your device. This information will be provided by your device manufacturer. You can install the application from the command line using the command:
+### Applications
 
-    snap install <application-name>
+Once `snap` is installed, you can easily install any driver. Find the name of the application compatible with your device. This information will be provided by your device manufacturer. You can install the application from the command line:
 
+    snap install <application-name>
 
+<br>
 *Note: Driver auto-installation i.e. when a printer on the system is discovered, the driver can be found on the Snap Store and even can be automatically installed using the hardware signature, is under development. We will update this work on our website as soon as this is completed. Until then you have to install drivers manually.*
+
+## Web Interface
+
+The embedded server can also provide a web interface to the Printer/Scanner Application, and PAPPL includes a standard web interface that can be customized and/or overridden. Aside from the usual status monitoring functionality, the web interface can be configured to allow remote users (with proper authentication) to:
+
+* Create and delete printers
+
+* Set the printer location and DNS-SD name
+
+* Configure the loaded media
+
+* Configure remote access accounts
+
+* Configure networking settings such as the hostname 
+
+* Update the TLS certificates used by the server
+
+![Webp net-resizeimage (1)](https://user-images.githubusercontent.com/43112419/97160934-1850f900-17a3-11eb-93cd-fe881c5816b7.png)
+
+___
+
+### Create and Delete Printers
+
+* **Create printers**
+
+    Resource path: /addprinter
+
+    ![Webp net-resizeimage (4)](https://user-images.githubusercontent.com/43112419/97161428-d2e0fb80-17a3-11eb-856b-b56752f4995e.png)
```

Comment:
Make sure that all content, including the images gets into our OpenPrinting web site repository.

Reply:
Ok, so do you want these images to be kept in the repository as well, as the images of the students?

Reply:
Yes, so that one can easily re-construct the whole site with a downloaded repo. External image storage places can go away and break the site. So best is when all the parts of the site are together at one place.